mock/src/mock/notes.rs

```diff
@@ -122,7 +122,7 @@ pub fn mock_notes(
             push.{created_note_2_tag}
             push.{created_note_2_asset}
             # MAST root of the `create_note` mock account procedure
-            call.0xeba65b5e6276e2c2f8e52f8ab8ce0c039bdfba127961826ff7d125c32d6d2458
+            call.0xbda092a816f2d979ecf67e0f39215c9f08b56823f41e3ad568db11dea63304e3
```

Comment:
How do you get to that MAST root?

Reply:
To get the MAST root we need to compile the module with this procedure. In this case, the procedure is defined in one of the test files, and the code that gives us the MAST root looks like this:
```Rust
let assembler = TransactionKernel::assembler();
let mut context = AssemblyContext::for_module(false);

let src = "use.miden::account
use.miden::tx
use.miden::contracts::wallets::basic->wallet

export.create_note
    # apply padding
    repeat.8
        push.0 movdn.9
    end

    # create note
    exec.tx::create_note
    # => [ptr, 0, 0, 0, 0, 0, 0, 0, 0]
end";

let module = ModuleAst::parse(src).unwrap();
let proc_roots = assembler.compile_module(&module, None, &mut context).unwrap();
println!("{}", proc_roots[0]);
```